src/main/java/racinggame/domain/RacingGame.java

```diff
@@ -0,0 +1,23 @@
+package racinggame.domain;
+
+import racinggame.dto.History;
+import racinggame.random.CapacityGenerator;
+
+public class RacingGame {
+
+    private Cars cars;
+    private Integer repeatCount;
```

Comment:
```suggestion
    private int repeatCount;
```

null을 다루지 않으므로 래퍼클래스는 불필요해보여요